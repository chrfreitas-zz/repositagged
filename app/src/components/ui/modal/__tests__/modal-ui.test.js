import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Modal from '../index';

describe('Modal view', () => {
  it('should match the snapshot with component default', () => {
    const props = {
      title: 'Modal',
      confirm: () => {},
      cancel: () => {},
    };

    const Children = () => 'Hello!';

    const renderer = new ShallowRenderer();
    const result = renderer.render(<Modal {...props}><Children /></Modal>);
    expect(result).toMatchSnapshot();
  });
});
