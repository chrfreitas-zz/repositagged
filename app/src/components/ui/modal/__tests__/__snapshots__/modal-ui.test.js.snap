// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal view should match the snapshot with component default 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className="modal"
  closeTimeoutMS={0}
  isOpen={false}
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  role="dialog"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
>
  <h5
    className="modal__title"
  >
    Modal
  </h5>
  <Children />
  <div
    className="modal__row"
  >
    <Button
      onClick={[Function]}
    >
      Save
    </Button>
    <Button
      onClick={[Function]}
    >
      Cancel
    </Button>
  </div>
</Modal>
`;
